Tests/BaseTestCase.php

```diff
@@ -27,15 +28,10 @@ public function setUp()
         $isDevMode = true;
         $paths = array(__DIR__ . "/Models");
 
-        $dbParams = array(
-            'driver' => 'pdo_mysql',
-            'user' => 'root',
-            'password' => '',
-            'dbname' => 'test',
-        );
+        $connectionConfig = $this->getConfig();
 
         $config = Setup::createAnnotationMetadataConfiguration($paths, $isDevMode);
-        $this->entityManager = DoctrineEntityManager::initialize($dbParams, $config);
+        $this->entityManager = DoctrineEntityManager::initialize($connectionConfig['parameters'], $config);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Managlea\Component\EntityManager\DoctrineEntityManager' in method 'setUp'.](https://www.codacy.com/app/Managlea/EntityManager/file/2304169728/issues/source?bid=3149133&fileBranchId=3194846#l34)